README

```diff
@@ -909,6 +912,9 @@ Prerequisites
     ithread support (introduced in Perl 5.6.0).
     <http://www.perl.org/>
 
+  * libpmwapi (optional)
+    Used by the `dcpmm` plugin.
+
```

Comment:
Could you please add the upstream URL for this library? That'll help others with getting it.

Reply:
This library is in the process of being open sourced and it should happen soon. I will add the URL once this is done.

Reply:
@mrunge I am trying to open source the dependent library by the end of next week and made available before Sept 30th. Please let me know if this could potentially be considered as a release candidate in that case?

Reply:
@htg1 Thank you for your work on this.
Please see https://mailman.verplant.org/pipermail/collectd/2019-August/007249.html

I'd rather move this to 5.11 then.